nomenclature/definitions/variable/economy/economy.yaml

```diff
@@ -4,6 +4,66 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Households|<Product>|Imported:
```

Comment:
Should there also be aggregate categories `Consumption|Households`, `Consumption|Government` and `Consumption|Industry`?

Reply:
To maintain the convention. I decided to add it.

Reply:
Thanks!